lib/ExpensesPage/expensesContainer.dart

```diff
@@ -0,0 +1,34 @@
+import 'package:flutter/material.dart';
+import 'package:percent_indicator/linear_percent_indicator.dart';
+import 'package:budgie/ExpensesPage/expensesPageClass.dart';
```

Comment:
I'm pretty sure you don't have to import using the full path if they are in the same folder

Reply:
I see, I'll look through your changes and update mine accordingly